### YamlMime:ManagedReference
items:
- uid: System.Transactions.TransactionInDoubtException
  id: TransactionInDoubtException
  children:
  - System.Transactions.TransactionInDoubtException.#ctor
  - System.Transactions.TransactionInDoubtException.#ctor(System.String)
  - System.Transactions.TransactionInDoubtException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Transactions.TransactionInDoubtException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: TransactionInDoubtException
  nameWithType: TransactionInDoubtException
  fullName: System.Transactions.TransactionInDoubtException
  type: Class
  summary: "不確定，請在交易上嘗試執行作業，或嘗試認可交易和交易時就會擲回的例外狀況會成為 InDoubt。"
  remarks: "在不確定的交易上嘗試執行動作時，會擲回這個例外狀況。 交易是不確定交易的狀態無法判斷。 具體來說，交易的最後結果是否認可或中止，永遠不會就此交易。       嘗試認可交易和交易成為 InDoubt 時，也會擲回這個例外狀況。       這是可復原的錯誤。"
  syntax:
    content: 'public class TransactionInDoubtException : System.Transactions.TransactionException'
  inheritance:
  - System.Exception
  - System.SystemException
  - System.Transactions.TransactionException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Transactions.TransactionInDoubtException.#ctor
  id: '#ctor'
  parent: System.Transactions.TransactionInDoubtException
  langs:
  - csharp
  name: TransactionInDoubtException()
  nameWithType: TransactionInDoubtException.TransactionInDoubtException()
  fullName: System.Transactions.TransactionInDoubtException.TransactionInDoubtException()
  type: Constructor
  assemblies:
  - System.Transactions
  namespace: System.Transactions
  summary: "初始化的新執行個體<xref href=&quot;System.Transactions.TransactionInDoubtException&quot;></xref>類別。"
  syntax:
    content: public TransactionInDoubtException ();
    parameters: []
  overload: System.Transactions.TransactionInDoubtException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Transactions.TransactionInDoubtException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Transactions.TransactionInDoubtException
  langs:
  - csharp
  name: TransactionInDoubtException(String)
  nameWithType: TransactionInDoubtException.TransactionInDoubtException(String)
  fullName: System.Transactions.TransactionInDoubtException.TransactionInDoubtException(String)
  type: Constructor
  assemblies:
  - System.Transactions
  namespace: System.Transactions
  summary: "初始化的新執行個體<xref href=&quot;System.Transactions.TransactionInDoubtException&quot;></xref>類別具有指定的訊息。"
  syntax:
    content: public TransactionInDoubtException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A<xref:System.String>包含說明例外狀況的發生原因的訊息。</xref:System.String>"
  overload: System.Transactions.TransactionInDoubtException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Transactions.TransactionInDoubtException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Transactions.TransactionInDoubtException
  langs:
  - csharp
  name: TransactionInDoubtException(SerializationInfo,StreamingContext)
  nameWithType: TransactionInDoubtException.TransactionInDoubtException(SerializationInfo,StreamingContext)
  fullName: System.Transactions.TransactionInDoubtException.TransactionInDoubtException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.Transactions
  namespace: System.Transactions
  summary: "初始化的新執行個體<xref href=&quot;System.Transactions.TransactionInDoubtException&quot;></xref>類別指定的序列化和串流內容資訊。"
  syntax:
    content: protected TransactionInDoubtException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "A <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref>說明失敗的序列化物件。"
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "A <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref>描述失敗的序列化內容的物件。"
  overload: System.Transactions.TransactionInDoubtException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Transactions.TransactionInDoubtException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Transactions.TransactionInDoubtException
  langs:
  - csharp
  name: TransactionInDoubtException(String,Exception)
  nameWithType: TransactionInDoubtException.TransactionInDoubtException(String,Exception)
  fullName: System.Transactions.TransactionInDoubtException.TransactionInDoubtException(String,Exception)
  type: Constructor
  assemblies:
  - System.Transactions
  namespace: System.Transactions
  summary: "初始化的新執行個體<xref href=&quot;System.Transactions.TransactionInDoubtException&quot;></xref>具有指定的訊息和內部例外狀況類別。"
  syntax:
    content: public TransactionInDoubtException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "A<xref:System.String>包含說明例外狀況的發生原因的訊息。</xref:System.String>"
    - id: innerException
      type: System.Exception
      description: "取得造成目前例外狀況的例外狀況執行個體。 如需詳細資訊，請參閱<xref:System.Exception.InnerException*>屬性。</xref:System.Exception.InnerException*>"
  overload: System.Transactions.TransactionInDoubtException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Transactions.TransactionException
  isExternal: false
  name: System.Transactions.TransactionException
- uid: System.Transactions.TransactionInDoubtException.#ctor
  parent: System.Transactions.TransactionInDoubtException
  isExternal: false
  name: TransactionInDoubtException()
  nameWithType: TransactionInDoubtException.TransactionInDoubtException()
  fullName: System.Transactions.TransactionInDoubtException.TransactionInDoubtException()
- uid: System.Transactions.TransactionInDoubtException.#ctor(System.String)
  parent: System.Transactions.TransactionInDoubtException
  isExternal: false
  name: TransactionInDoubtException(String)
  nameWithType: TransactionInDoubtException.TransactionInDoubtException(String)
  fullName: System.Transactions.TransactionInDoubtException.TransactionInDoubtException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Transactions.TransactionInDoubtException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Transactions.TransactionInDoubtException
  isExternal: false
  name: TransactionInDoubtException(SerializationInfo,StreamingContext)
  nameWithType: TransactionInDoubtException.TransactionInDoubtException(SerializationInfo,StreamingContext)
  fullName: System.Transactions.TransactionInDoubtException.TransactionInDoubtException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Transactions.TransactionInDoubtException.#ctor(System.String,System.Exception)
  parent: System.Transactions.TransactionInDoubtException
  isExternal: false
  name: TransactionInDoubtException(String,Exception)
  nameWithType: TransactionInDoubtException.TransactionInDoubtException(String,Exception)
  fullName: System.Transactions.TransactionInDoubtException.TransactionInDoubtException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Transactions.TransactionInDoubtException.#ctor*
  parent: System.Transactions.TransactionInDoubtException
  isExternal: false
  name: TransactionInDoubtException
  nameWithType: TransactionInDoubtException.TransactionInDoubtException
